presto-docs/src/main/sphinx/installation/deploy-docker.rst

```diff
@@ -0,0 +1,116 @@
+=================================
+Deploy Presto From a Docker Image
+=================================
+
+These steps were developed and tested on Mac OS X, on both Intel and Apple Silicon chips. 
+
+Following these steps, you will:
+
+- install the command line tools for brew, docker, and `colima <https://github.com/abiosoft/colima>`_ tools
+- verify your Docker setup
+- pull the Docker image of the Presto server and the Presto CLI
+- start your local Presto server and Presto CLI
+- query your local Presto server using the Presto CLI
+- query a remote Presto server using the Presto CLI
+
+Installing brew, docker, docker-compose, and colima
+===================================================
+
+This task shows how to install brew, then to use brew to install docker, docker-compose, 
+and colima. 
+
+Note: If you have Docker installed you can skip steps 1-3. It is recommended that you 
+verify your Docker setup by running the command in step 4.
+
+1. If you do not have brew installed, run the following command:
+
+   ``/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"``
+
+2. To install the docker command line and `colima <https://github.com/abiosoft/colima>`_ tools, run the following command:
+
+   ``brew install docker docker-compose colima``
+
+3. Run the following command: 
+
+   ``colima start``
+
+4. To verify your local setup, run the following command:
+
+   ``docker run hello-world``
+
+   If you see a response similar to the following, you are ready.
+
+   ``Hello from Docker!`` 
+   ``This message shows that your installation appears to be working correctly.``
+
+Installing and Running the Presto Docker container
+==================================================
+
+1. Run the following command to download the prestodb-sandbox container from `DockerHub <https://hub.docker.com/r/ahanaio/prestodb-sandbox>`_: 
+
+   ``docker pull ahanaio/prestodb-sandbox``
+
+   Downloading the container may take a few minutes. When the download completes, go on to the next step.
+
+2. To start the Presto server in the Docker container, run the following command:
+
+   ``docker run -p 8080:8080 --name presto ahanaio/prestodb-sandbox``
+
+   This command also assigns the name ``presto`` for ``ahanaio/prestodb-sandbox``.
+
+   If you see a response similar to the following, the Presto server is running in the Docker container.
```

Comment:
NIT Since logging seems to be set to "info" level by default, it may be useful to have something here about it logging some startup info before the `SERVER STARTED` message is received

```suggestion
  The Presto server begins logging startup information in the terminal window. Once you see a response similar to the following, the Presto server is running in the Docker container.
```